.ci/no_old_refs.sh

```diff
@@ -1,35 +1,29 @@
 #!/bin/bash
-# Attention, there is no "-x" to avoid problems on Travis
-set -e
```

Comment:
why you removed ability to fail fast (fail on first problem) ?
without this you will always deal with NOT root reasons of failure.

you can remove comment, but `set -e` is useful.

Reply:
there is no need for it in github actions, it fails when exit code is not zero.

I reverted this change

Reply:
good, but this shell script one day be execution in other CI, where it required to fail fast.
My local always need `set -e`.